user-interface/src/main/java/life/qbic/datamanager/views/navigation/ProjectSideNavigationComponent.java

```diff
@@ -220,22 +237,90 @@ private static SideNavItem createProjectUsers(String projectId) {
         VaadinIcon.USERS.create());
   }
 
-  private static Div createExperimentSection(String projectId, List<Experiment> experimentsList) {
+  private Div createExperimentSection(String projectId, List<Experiment> experimentsList) {
     Div experimentSection = new Div();
-    SideNavItem experiments = new SideNavItem("");
-    experiments.setLabel("EXPERIMENTS");
-    experiments.setPrefixComponent(VaadinIcon.FLASK.create());
+    SideNavItem expHeader = new SideNavItem("EXPERIMENTS");
+    Icon flask = VaadinIcon.FLASK.create();
+    flask.addClassName("primary");
+    expHeader.setPrefixComponent(flask);
+    expHeader.addClassName("primary");
```

Comment:
If you want the experiment icon to be blue when the user is within an experiment there is an easy work around as follows: 
```suggestion
    //The icon should appear blue if the user is within an experiment
    if(context.experimentId().isPresent()){
      flask.setClassName("primary");
    }
```